action_id: trunk-upgrade-available
date: 2025-04-17 08:36:48
run: trunk upgrade --notify --allow-daemon-start=false
run_from: ""
trigger: schedule 1h
result:
  exit_code: 1
  out: ""
  err: Exception occurred while executing 'trunk-upgrade-available'. See .trunk\out\2kSJa.yaml for details.
  error_path: D:\box\ODSLY\ODSLY\.trunk\out\2kSJa.yaml
timestamp: 1744879008
full_timestamp:
  nanos: 172051000
  seconds: 1744879008
trunk_version: 1.22.12
